### ISSUE TITLE
The Omnilock of ForceBridge

### ISSUE THREAD

**Issue by Daryl-L** [Magickbase/ckb-explorer-public-issues#532]
## History
The Yokai Swap launched in 2022, but at that time, the Force Bridge was using the PW-Lock, which was not supported by Yokai Swap. But at that time, the Omnilock hadn't launched in Mainnet. So we just deployed the Omnilock from the source code which had not been released. And the [migration](https://github.com/nervosnetwork/force-bridge/blob/main/docs/asset-migration-guide.md) from PW-Lock was also launched for this.

And the Yokai Swap and the Force Bridge are still using this version of Omnilock now.

## Dependency
```
OMNI_LOCK: {
  CODE_HASH: "0x9f3aeaf2fc439549cbc870c653374943af96a0658bd6b51be8d8983183e6f52f",
  HASH_TYPE: "type",
  TX_HASH: "0xaa8ab7e97ed6a268be5d7e26d63d115fa77230e51ae437fc532988dd0c3ce10a",
  INDEX: "0x1",
  DEP_TYPE: "code",
}
```

## Refference
https://forcebridge.com
https://github.com/homura/pw-up/blob/master/src/lib/PwUp.ts
https://www.yokaiswap.com/bridge/deposit
https://pw-up.vercel.app/
https://github.com/nervosnetwork/force-bridge/blob/main/docs/asset-migration-guide.md

**Comment by homura** [comment]
Is the issue asking the explorer team to add the outdated Omnilock to its known scripts list?

**Comment by Keith-CY** [comment]
> Is the issue asking the explorer team to add the outdated Omnilock to its known scripts list?

Yes, this script is used by several dapps, and it was declared an omnilock, but in fact it's a customized one, so it would be better to be annotated.

**Comment by Keith-CY** [comment]
PR: https://github.com/nervosnetwork/ckb-explorer-frontend/pull/1447

**Comment by Keith-CY** [comment]
The PR will be merged first, the link to the code will be added after the Chinese New Year.

The link to the code should point to the git commit which was deployed on the chain.

**Comment by Keith-CY** [comment]
Force Bridge Lock has been explicitly marked at https://explorer.nervos.org/script/0x9f3aeaf2fc439549cbc870c653374943af96a0658bd6b51be8d8983183e6f52f/type
